### PR TITLE
travis: enable vm jobs of aarch64

### PR DESCRIPTION
**Pull request by Jianlin-lv**
Current Arm64 job often gets the following issue:
    "No output has been received in the last 10m0s"
Perhaps it can be avoided by enabling the VM job of arm64.

Signed-off-by: Jianlin Lv <Jianlin.Lv@arm.com>

### CODE REVIEW

**Review by pchaigno — APPROVED** [review]
Wow, I didn't know that was already available! I'm going to need to enable it for strace too :-)

Looks like the build is also a lot faster on this new environment :tada:

**Comment by Jianlin-lv** [comment]
> Wow, I didn't know that was already available! I'm going to need to enable it for strace too :-)
> 
> Looks like the build is also a lot faster on this new environment 🎉

As far as I know, Travis will publicly announce AWS Graviton2 support soon.
Hope this change can make the Arm64 job more stable.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 go: "1.15"
 
-arch:
-  - amd64
-  - arm64
 
 jobs:
+  include:
+    - arch: amd64
+      dist: bionic
```
Comment (sayboras) [inline .travis.yml]: I just noticed recentlly, that the cilium runtime is with focal. Not sure if we can upgrade amd64 dist to focal here.

```shell script
$ docker run -it cilium/cilium:latest cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

**Review by sayboras — APPROVED** [review]
LGTM, one small question.

File: .travis.yml (hunk 1 above)
Comment (sayboras) [reply .travis.yml]: and vagrant dev VM is with bionic right now as well :thinking:

Comment (Jianlin-lv) [reply .travis.yml]: Focal is available for Amd64 job; 
Good comments, Thanks.

**Review by kkourt — APPROVED** [review]

**Pull request merged**